docs/antora-ccutil/antora-playbook-ccutil.yml

```diff
@@ -0,0 +1,57 @@
+site:
+  title: Keycloak Docs
+  # the 404, site-map, and canonical urls are only generated when an url is defined
+  url: http://localhost:5000/keycloak-antora/
+  start_page: guides-server::index.adoc
+content:
+  sources:
+    - url: ../..
+      start_paths:
+        - docs/antora-ccutil/components/guides-server
+        - docs/antora-ccutil/components/guides-operator
+      branches:
+        - 'HEAD'
+
+urls:
+  latest_version_segment: latest
+  html_extension_style: drop
+
+antora:
+  extensions:
+    - '@antora/collector-extension'
+    - './lib/index.js'
+
+ui:
+  bundle:
+    url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/HEAD/raw/build/ui-bundle.zip?job=bundle-stable
+    # snapshot: true
+  supplemental_files: supplemental_ui
+  output_dir: antora
+
+runtime:
+  fetch: true
+  cache_dir: target/antora/cache
+  log:
+    failure_level: error
+    format: pretty
+
+output:
+  clean: true
+  dir: target/site/keycloak-antora
+
+# Global asciidoc attributes here, used across all versions of documentation
+asciidoc:
+  attributes:
+    idprefix: ''
+    idseparator: '-'
+    experimental: ''
+    page-pagination: ''
+    project_name: Keycloak
+    project_name_full: Keycloak
+    project_version: 20.0.0
+    project_versionMvn: 20.0.0
```

Comment:
In product docs these need to somehow include the `-redhat` suffix. Not sure how we'll do that as the sources for `RHBK` doesn't override this version in the file, but PNC just overrides it somehow I think. @ASzc

Reply:
Actually, if we make PNC build the keycloak-guides POM as well, then that'll have the correct versions right?

Reply:
Perhaps we could add a versions.adoc or something to the keycloak-guides that have the version properties templated in through Maven?

Reply:
Will be migrated to a `versions.adoc` in a different PR, will keep it as is in this PR.